src/plugins/twitter/twitter.js

```diff
@@ -31,6 +31,61 @@ var componentName = "wb-twitter",
 			protocol = wb.pageUrlParts.protocol;
 
 		if ( eventTarget ) {
+
+			// Ignore IE11
+			// Note: Twitter's widget script doesn't render anything in IE11... so there's no value in showing an endless loading icon in it
+			if ( !wb.ie11 ) {
+				const linkSelector = "a.twitter-timeline",
+					twitterLinks = eventTarget.querySelectorAll( linkSelector ),
+					loadingClass = "twitter-timeline-loading";
+				let loadingRemovals = 0,
+					observer;
+
+				// Insert loading icons below Twitter links
+				twitterLinks.forEach( function( twitterLink ) {
+					const loadingDiv = document.createElement( "div" );
```

Comment:
```suggestion
					let loadingDiv = document.createElement( "div" );
```

`const` in this context are not sealing the inner object.

Reply:
It isn't supposed to :P.

I felt like ``const`` was a better bet than ``let`` for this since the ``div`` element that's getting created isn't intended to turn into a different element later on. Only its properties are subject to change (like getting a class name).

As we chatted about yesterday, I could change this to ``let`` if you want since that's more in line with what most programming languages usually expect a constant to represent (i.e. set-in-stone value). But I feel like that would betray how JavaScript ``const`` variables are meant to work :D lol.

I know that ``const`` is kind of quirky in JavaScript since it doesn't require DOM node properties to be totally set-in-stone. But at the same time, IMO it would be inappropriate to forego using ``const`` in those quirky situations just to follow a common convention that only applies to *other* programming languages.

Reply:
Even though I disagreed with what was being asked here, I ultimately ended changing ``const`` to ``let`` for ``loadingDiv`` due to other unrelated changes.

``forEach()`` is no longer being used at this spot... Each plugin instance is now expected to have only one loading icon at most.

``loadingDiv`` is now declared at the start of the plugin since it's now being used for both adding and removing the loading icon.

I could've technically kept using ``const``, but a loading icon only gets added if a suitable Twitter link exists... so there's no point in creating a useless HTML element for ``loadingDiv`` unless a link is actually found. So I went with ``let`` since it's more flexible.